<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_Popup_ListEdit_Numbers</class>
 <widget class="QDialog" name="D_Popup_ListEdit_Numbers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit_Numbers">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_Update">
     <property name="text">
      <string>Update (F5)</string>
     </property>
     <property name="shortcut">
      <string>F5</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter one number per line</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_Ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4" colspan="2">
    <layout class="QGridLayout" name="gridLayout_NumbersPlot">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_Ok</tabstop>
  <tabstop>pushButton_Update</tabstop>
  <tabstop>plainTextEdit_Numbers</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
